packages/trip-form/src/TripOptions/Checkbox.tsx

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import * as S from "./styled";
+import TrimetModeIcon from "../../../icons/lib/trimet-mode-icon-2021";
+
+// FIXME: Move this to @opentripplanner/types added in https://github.com/opentripplanner/otp-ui/pull/281
+export interface ButtonProps {
+  onClick(): void;
+  selected: boolean;
+}
+
+export default function Checkbox(
+  props: {
+    checked: boolean;
+    children: React.ReactNode | string;
+    className?: string;
+    inset?: boolean;
+    mode?: string;
+  } & ButtonProps
+): React.ReactElement {
+  const {
+    checked,
+    children,
+    className,
+    inset,
+    mode,
+    onClick,
+    selected
+  } = props;
+  return (
+    <S.Checkbox
+      className={`${className || ""} ${inset ? "inset" : ""}`}
+      onClick={onClick}
+      selected={selected}
```

Comment:
See comment for turning `<S.Checkbox>` to be a styled `<label>`. For semantics and a11y, add an `<input type="checkbox">` as a child, made invisible in the CSS for `S.Checkbox`,
and move these two props to the checkbox element.

Reply:
As above, the semantic changes create unexpected css changes. Have added aria labels and roles which pass Firefox's a11y inspector in b83ec0d22f6a9cc4db839bfe11bb409c9cbfbb38